File: Programming-languages-and-laboratory/Python/Exercises/36_compito_python_20200116/Visita.py
class Visita:
    def __init__(self, codice_supermercato, importo_speso):
        self._codice_supermercato = codice_supermercato
        self._importo_speso = importo_speso

    def get_importo_speso(self):
        return self._importo_speso

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/36_compito_java_2020_01_16/Visita.java
public class Visita {
	protected int codiceSupermercato;
	protected float importoSpeso;
	
	public Visita(int codiceSupermercato, float importoSpeso) {
		this.codiceSupermercato = codiceSupermercato;
		this.importoSpeso = importoSpeso;
	}
	
	public int getCodiceSupermercato() {
		return codiceSupermercato;
	}
}